packages/core/src/build-time-plugins/buildTimeOptionsBase.ts

```diff
@@ -84,6 +84,23 @@ export interface BuildTimeOptionsBase {
    */
   silent?: boolean;
 
+  /**
+   * When an error occurs during release creation or sourcemaps upload, the plugin will call this function.
+   *
+   * By default, the plugin will simply throw an error, thereby stopping the bundling process.
```

Comment:
Unless I'm mistaken, by default the plugin fails gracefully by printing the error but not exiting the build process.

Reply:
The docs are taken from here: https://www.npmjs.com/package/@sentry/vite-plugin#errorhandler

If this is not correct, we should also fix it in the bundler-plugins readme.